### PR TITLE
Fixed dropdown

### PR DESCRIPTION
**Pull request by igorkorsukov**
some issues

### CODE REVIEW

File: src/framework/uicomponents/qml/MuseScore/UiComponents/Dropdown.qml [hunk 1]
```diff
@@ -67,18 +68,43 @@ Item {
...
 
-            root.currentText = accesser.itemAt(root.currentIndex).text
-            root.currentValue = accesser.itemAt(root.currentIndex).value
+            root.currentText = root.valueFromModel(root.currentIndex, root.textRole, "")
+            root.currentValue = root.valueFromModel(root.currentIndex, root.valueRole, "")
         }
     }
 
+    function valueFromModel(index, roleName, def) {
+
```
Comment (Eism) [inline src/framework/uicomponents/qml/MuseScore/UiComponents/Dropdown.qml:77]: please remove these two empty lines

File: src/framework/uicomponents/qml/MuseScore/UiComponents/Dropdown.qml [hunk 2]
```diff
@@ -21,6 +21,7 @@
  */
 import QtQuick 2.15
 import QtQuick.Controls 2.15
+import QtQml.Models 2.2
```
Comment (Eism) [inline src/framework/uicomponents/qml/MuseScore/UiComponents/Dropdown.qml:24]: 2.15

File: src/framework/uicomponents/qml/MuseScore/UiComponents/Dropdown.qml (hunk 1 above)
Comment (igorkorsukov) [reply src/framework/uicomponents/qml/MuseScore/UiComponents/Dropdown.qml]: fixed

File: src/framework/uicomponents/qml/MuseScore/UiComponents/Dropdown.qml (hunk 2 above)
Comment (igorkorsukov) [reply src/framework/uicomponents/qml/MuseScore/UiComponents/Dropdown.qml]: fixed

**Review by Eism — APPROVED** [review]

**Pull request merged**